test/lib/testAttributeEditorTemplate.py

```diff
@@ -321,12 +367,35 @@ def testAECustomEnumControl(self):
         self.assertTrue(cmds.formLayout(shaderFormLayout, exists=True))
         startLayout = cmds.formLayout(shaderFormLayout, query=True, fullPathName=True)
         self.assertIsNotNone(startLayout, 'Could not get full path for Shader formLayout')
-        
+
+        frameLayout = self.searchForMayaControl(startLayout, cmds.frameLayout, 'Shader: glTF PBR')
+        self.assertIsNotNone(frameLayout, 'Could not find "Shader: glTF PBR" frameLayout')
+
+        self.expandFrameLayout(frameLayout, fullPrimPath)
+
+        shaderFormLayout = self.attrEdFormLayoutName('Shader')
```

Comment:
For nested sections, we need to search the section from the top again because they may have changed names ("frame28" to "frame31" for example).